gateway/tests/test_utils.py

```diff
@@ -129,3 +129,10 @@ def test_get_swagger_urls(self):
         for module in modules:
             assert module.endpoint_name in urls
             assert urls[module.endpoint_name] == get_swagger_url_by_logic_module(module)
+
+
+class TestUnavailableLogicModule(TestCase):
+    @patch('requests.get')
+    def test_unavailable_logic_module(self, mock_request_get):
+        mock_request_get.side_effect = [ConnectionError, TimeoutError]
```

Comment:
The function `requests.get` is called only once, so only the first exception will be raised. Therefore, we need to implement two tests to test the different exceptions.
https://docs.python.org/3/library/unittest.mock-examples.html#side-effect-functions-and-iterables